german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Wo ist das denn ein elitärer Sprachgebrauch. Das ist halt die Bezeichnung dafür. Diesen Panik vor Fachbegriffen Einwurf kannst du bei so vielen Dingen bringe, dass quasi kein einziges Wort mehr für irgendwas zu nutzen wäre. Dann dürftest du auch nicht von einem Zweihänder, Langschwert, Kettenhemd usw sprechen. Das sind alles genauso elitäre Begriffe.
Unvoreingenommene Spieler ebenso Neulinge werden bei Begriffen, die sie nicht kennen Google verwenden und dann direkt den Eintrag bei Wikipedia vorfinden und wissen dann um was es sich handelt. Wie bei den meisten Begriffen, die einem erstmal so nichts sagen.
Noch schlimmer wird es ja dann wenn Zaubersprüche mit ins Spiel kommen, das ist ja noch ein größerer Schritt in die In-Group. In dem Moment, in dem man mit dem Hobby anfängt gehört man dazu. Ob man will oder nicht.^^